setup.py

```diff
@@ -21,7 +21,6 @@
     "aiofiles",
     "apispec>=1.0.0b1",
     "chardet",
-    "docopt-ng",
```

Comment:
Is there a reason why we are removing `docopt-ng`? I know its being used in the `cli.py` file.

Reply:
`cli.py` did not exist in this project as I arrived. I asked Kenneth about it, but it seemed he also didn't know. Maybe it fell victim of the reverts he was doing to the code base?

Reply:
In this spirit, the dependency can be re-added when we re-add `cli.py` again. Maybe we can find it back in some previous commits or PRs?

Reply:
> Maybe we can find it back?

Here we go:

- GH-72
- GH-104

Reply:
Let's have a dedicated issue to track this regression.

- GH-547